SeaPublicWebsite/Services/QuestionFlowService.cs

```diff
@@ -777,6 +791,7 @@ public enum QuestionFlowPage
         ServiceUnsuitable,
         AskForPostcode,
         ConfirmAddress,
+        NoAddressFound,
```

Comment:
These two are essentially identical for all purposes. Both pages are handled by the same 'get' method. Should I just create one `Address` `QuestionFlowPage`?

Reply:
As discussed we'll remove the separate "not found" view so these will actually be the same page